.github/workflows/testing.yml

```diff
@@ -26,29 +26,32 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@v2
 
-      - name: Build Docker base image
-        uses: docker/build-push-action@v1
+      - name: Login to DockerHub
+        uses: docker/login-action@v1 
         with:
           username: ${{ secrets.DOCKER_USERNAME }}
-          password: ${{ secrets.DOCKER_PASSWORD }}
-          repository: hellofreshtech/kangal-jmeter
-          path: docker/jmeter-base/.
+          password: ${{ secrets.DOCKER_TOKEN }}
+
```

Comment:
This can be removed since we're simply building the docker images and not pushing them